### PR TITLE
[batch] index jobs table on state

### PR DESCRIPTION
**Pull request by cseed**

### CODE REVIEW

File: batch/create-batch-tables.sql [hunk 1]
```diff
@@ -40,6 +40,7 @@ CREATE TABLE IF NOT EXISTS `jobs` (
   PRIMARY KEY (`batch_id`, `job_id`),
   FOREIGN KEY (`batch_id`) REFERENCES batch(id) ON DELETE CASCADE
 ) ENGINE = InnoDB;
+CREATE INDEX jobs_state ON `jobs` (state);
```
Comment (jigold) [inline batch/create-batch-tables.sql]: Put state in backticks. Might be a SQL reserved word.

**Comment by jigold** [comment]
@danking thinks we should also put it on deleted in the batch table.

**Comment by cseed** [comment]
And added backticks

File: batch/create-batch-tables.sql [hunk 2]
```diff
@@ -41,7 +41,7 @@ CREATE TABLE IF NOT EXISTS `jobs` (
   PRIMARY KEY (`batch_id`, `job_id`),
   FOREIGN KEY (`batch_id`) REFERENCES batch(id) ON DELETE CASCADE
 ) ENGINE = InnoDB;
-CREATE INDEX jobs_state ON `jobs` (state);
+CREATE INDEX `jobs_state` ON `jobs` (`state`);
```
Comment (jigold) [inline batch/create-batch-tables.sql]: Do you need an IF EXISTS?

Comment (cseed) [reply batch/create-batch-tables.sql]: No, create_batch_tables only runs the SQL if the tables don't already exist (using jobs as a proxy): https://github.com/hail-is/hail/blob/master/batch/create-batch-tables-pod.yaml#L15

**Pull request merged**